buildspec.yml

```diff
@@ -1,34 +1,30 @@
-version: 0.1
-environment_variables:
-  plaintext:
-    BUILD_PHASE_MARKER: build_phase_complete
+version: 0.2
+env:
+  variables:
+    PRX_LAMBDA_CODE_S3_KEY: "lambda/PRX-cms-image-lambda.zip"
+    PRX_LAMBDA_CODE_CONFIG_PARAMETERS: "CmsImageLambdaCodeS3ObjectVersion"
+    PRX_LAMBDA_ARCHIVE_BUILD_PATH: "/app/build.zip"
+  parameter-store:
+    CMS_ACCESS_KEY_ID: "/prx/test/cms-image-lambda/AWS_ACCESS_KEY_ID"
+    CMS_DEFAULT_REGION: "/prx/test/cms-image-lambda/AWS_DEFAULT_REGION"
+    CMS_SECRET_ACCESS_KEY: "/prx/test/cms-image-lambda/AWS_SECRET_ACCESS_KEY"
+    CODECOV_TOKEN: "/prx/test/cms-image-lambda/CODECOV_TOKEN"
 phases:
-  install:
-    commands:
-      - apt-get update -y
-      - apt-get install -y jq
-  pre_build:
-    commands:
-      - npm install
   build:
     commands:
-      - npm test
-      - touch $BUILD_PHASE_MARKER
+      - "cd $(ls -d */|head -n 1)"
+      - "cp env-example .env"
+      - "echo AWS_ACCESS_KEY_ID=$CMS_ACCESS_KEY_ID >> .env"
+      - "echo AWS_DEFAULT_REGION=$CMS_DEFAULT_REGION >> .env"
+      - "echo AWS_SECRET_ACCESS_KEY=$CMS_SECRET_ACCESS_KEY >> .env"
+      - "echo CODECOV_TOKEN=$CODECOV_TOKEN >> .env"
+      - "echo CODEBUILD_BUILD_ID=$CODEBUILD_BUILD_ID >> .env"
+      - "echo PRX_REPO=$PRX_REPO >> .env"
+      - "echo PRX_COMMIT=$PRX_COMMIT >> .env"
+      - "echo PRX_BRANCH=$PRX_BRANCH >> .env"
+      - "echo PRX_GITHUB_PR=$PRX_GITHUB_PR >> .env"
```

Comment:
Why do all the echos instead of just listing the vars in the docker-compose file?

Reply:
I actually need the ENVs set inside the docker container, inside the other docker container.

So if I list/set them here, they end up defined in the host container, but not the child container (running the tests).